pyproject.toml

```diff
@@ -12,7 +12,7 @@
   maintainers = [{name = "David Bonnes", email = "zxdavb@bonnes.me"}]
   dependencies = [
     "colorlog>=6.8.2",
-    "paho-mqtt>=1.6.1",
+    "paho-mqtt==2.1.0",  # fixes deprecated V1 protocol warning
```

Comment:
Never pin a version in a client library - that is up to the app (i.e. Home Assistant).

have `>=` and only that if possible (i.e. no <).

Reply:
Rebase you other PRs and I'll merge them.